lib/ecto/migration.ex

```diff
@@ -781,6 +797,13 @@ defmodule Ecto.Migration do
     Runner.flush
   end
 
+  @doc """
+  """
+  def view(view_name), do: %View{name: view_name}
```

Comment:
We should probably remove this clause if we don't don't support views without queries.